cfme/infrastructure/config_management.py

```diff
@@ -665,14 +673,11 @@ def step(self):
 
 @navigator.register(ConfigSystem, 'All')
 class SysAll(CFMENavigateStep):
-    VIEW = ConfigManagementAllView
+    VIEW = ConfigSystemAllView
     prerequisite = NavigateToAttribute('appliance.server', 'LoggedIn')
 
     def step(self):
         self.prerequisite_view.navigation.select('Configuration', 'Management')
```

Comment:
Q: why there are no steps for Ansible Tower view?

Reply:
Not sure on that one, there weren't any steps before either.